src/pages/library/DataBrowserToggler.js

```diff
@@ -1,15 +1,83 @@
+import _ from 'lodash/fp'
+import { Fragment, useState } from 'react'
 import { div, h, label, strong } from 'react-hyperscript-helpers'
-import { Link, Switch } from 'src/components/common'
+import { ButtonPrimary, IdContainer, Link, spinnerOverlay, Switch } from 'src/components/common'
+import { icon } from 'src/components/icons'
+import { TextArea, TextInput } from 'src/components/input'
+import Modal from 'src/components/Modal'
+import TooltipTrigger from 'src/components/TooltipTrigger'
+import { Ajax } from 'src/libs/ajax'
 import colors from 'src/libs/colors'
 import { isDataBrowserVisible } from 'src/libs/config'
+import { FormLabel } from 'src/libs/forms'
 import * as Nav from 'src/libs/nav'
 import { useStore } from 'src/libs/react-utils'
 import { authStore } from 'src/libs/state'
+import * as Utils from 'src/libs/utils'
 import { catalogPreviewStore } from 'src/pages/library/dataBrowser-utils'
+import validate from 'validate.js'
+
+
+const DataBrowserFeedbackModal = ({ onSuccess, onDismiss }) => {
+  const [contactEmail, setContactEmail] = useState('')
+  const [feedback, setFeedback] = useState('')
+  const [submitting, setSubmitting] = useState(false)
+
+  const submit = Utils.withBusyState(setSubmitting, async () => {
+    await Ajax().Surveys.submitForm('1FAIpQLSevEVLKiLNACAsti8k2U8EVKGHmQ4pJ8_643MfdY2lZEIusyw',
```

Comment:
That's good idea, although I don't think we need to worry about too much feedback from different places. It seems unlikely that we'll get feedback from dev that wouldn't be applicable for prod as well.

We did think about making the form elements configurable, and that way could be different in different environments, but it doesn't seem likely that this would be needed.